app/views/foods/_food.html.erb

```diff
@@ -1,17 +1,12 @@
 <div id="<%= dom_id food %>">
-  <p>
-    <strong>Name:</strong>
-    <%= food.name %>
-  </p>
-
-  <p>
-    <strong>Measurement unit:</strong>
-    <%= food.measurement_unit %>
-  </p>
-
-  <p>
-    <strong>Price:</strong>
-    <%= food.price %>
-  </p>
-
+  <tr>
+    <td><%= food.name %></td>
+    <td><%= food.measurement_unit %></td>
+    <td><%= food.price %></td>
```

Comment:
Price should be in USD it seems, according to the project wireframe.

I pelive you could do something like
```
 <td><%= "$#{food.price}" %></td>
```

This way the price would show as $1 instead of just 1, for example